scheds/rust/scx_chaos/src/bpf/main.bpf.c

```diff
@@ -452,6 +463,21 @@ void BPF_STRUCT_OPS(chaos_enqueue, struct task_struct *p __arg_trusted, u64 enq_
 	    enqueue_chaotic(p, enq_flags, taskc))
 		return;
 
+	// NOTE: this may not work for affinitized tasks because p2dq does
+	// direct dispatch in some situations.
+	if (taskc->next_trait == CHAOS_TRAIT_DEGRADATION && degradation_percent > 0) {
```

Comment:
nit: the `> 0` shouldn't be necessary here either as it'll just be `* 100 / 100`.